koalabot.py

```diff
@@ -149,25 +150,32 @@ async def on_command_error(ctx, error: Exception):
     else:
         guild_id = ctx.guild.id
 
-    if error.__class__ in [commands.MissingRequiredArgument,
-                           commands.CommandNotFound]:
+    if error.__class__ in [commands.MissingRequiredArgument]:
+        await ctx.send(embed=error_embed(error_type=str(type(error).__name__),
+                                         description=str(error)+"\nAn argument is missing from this command. "
+                                                                "Use k!help <command> for more information.")) #TODO: Make <command> actually change to the command inputted
+    if error.__class__ in [commands.CommandNotFound]:
         await ctx.send(embed=error_embed(description=error))
     if error.__class__ in [commands.CheckFailure]:
         await ctx.send(embed=error_embed(error_type=str(type(error).__name__),
-                                         description=str(error)+"\nPlease ensure you have administrator permissions, "
-                                                                "and have enabled this extension."))
+                                         description=str(error)+"\nThe command could not be executed with the given "
+                                                                "parameter."))

```

Comment:
I don't believe this change helps convey the issue very well, it seems to be a bit too abstract.